master/internal/checkpoints/postgres_checkpoints.go

```diff
@@ -54,25 +54,40 @@ func GetModelIDsAssociatedWithCheckpoint(ctx context.Context, ckptUUID uuid.UUID
 	return modelIDs, nil
 }
 
+// ModelInfo is a struct containing info used for locating models.
+type ModelInfo struct {
+	ID      int
+	Version int
+	Name    string
+}
+
 // GetRegisteredCheckpoints gets the checkpoints in
 // the model registrys from the list of checkpoints provided.
-func GetRegisteredCheckpoints(ctx context.Context, checkpoints []uuid.UUID) (map[uuid.UUID]bool, error) {
+func GetRegisteredCheckpoints(ctx context.Context, checkpoints []uuid.UUID) (map[uuid.UUID]ModelInfo, error) {
```

Comment:
Is the user's RBAC permission scope being check against the model/model versions where this is being used? I'm glad we're communicating this, I just want to make sure we don't end up exposing any information unintentionally.

Reply:
yes -- we filter the model list [here](https://github.com/determined-ai/determined/pull/9178/files#diff-dbc2f198f92247702ba512417755edb5870aa87af786002cfe609abbf7c7e9caR339-R348)

Reply:
While that accounts for it there, it doesn't account for it in
`PatchCheckpoints` where `registeredCheckpointUUIDs` is being passed into the error response.